src/cmd/setup-tachyon.js

```diff
@@ -147,8 +153,8 @@ module.exports = class SetupTachyonCommands extends CLICommandBase {
 				ui: this.ui,
 			}));
 		} catch (error) {
-			// ignore error and return default values
-			this.ui.write('We couldn\'t get the device info.');
+			// If this fails, the flash won't work so abort early.
+			throw new Error('Unable to get device info. Please restart the device and try again.');
```

Comment:
If I Ctrl-C the flash and start it again, I get the warning here then printing the device info crashes with "cannot read properties of undefined reading deviceId" from _printDeviceInfo so I think the error is fatal.